nano/node/active_transactions.cpp

```diff
@@ -310,15 +333,60 @@ void nano::active_transactions::request_loop ()
 	{
 		request_confirm (lock);
 		update_active_difficulty (lock);
+		const auto extra_delay (std::min (roots.size (), max_broadcast_queue) * node.network.broadcast_interval_ms * 2);
+		condition.wait_for (lock, std::chrono::milliseconds (node.network_params.network.request_interval_ms + extra_delay));
+	}
+}
 
-		// This prevents unnecessary waiting if stopped is set in-between the above check and now
-		if (stopped)
+void nano::active_transactions::prioritize_frontiers_for_confirmation (std::chrono::milliseconds time_a)
```

Comment:
There is already confirm_frontiers transaction_a, may be use it?

Reply:
makes sense :)